.SRCINFO

```diff
@@ -43,24 +43,38 @@ pkgbase = electron-ozone
 	conflicts = electron
 	options = debug
 	options = !strip
-	source = git+https://github.com/hedgepigdaniel/electron.git#branch=arch7.1.8-1
+	source = git+https://github.com/vially/electron.git#branch=arch8.2.1-1
```

Comment:
I think it would be better if this was linking to your fork to keep things simpler (e.g.: less forks).

I would have created a pull-request for that too but it's not possible through GitHub because branch `arch8.2.1-1` does not yet exist in your fork. So it might be easier if you'd just copy this branch from my fork and then this could become:
```suggestion
	source = git+https://github.com/hedgepigdaniel/electron.git#branch=arch8.2.1-1
```